pkg/api/persistentvolume/util_test.go

```diff
@@ -134,10 +134,10 @@ func TestDropDisabledFields(t *testing.T) {
 
 			DropDisabledFields(tc.newSpec, tc.oldSpec)
 			if !reflect.DeepEqual(tc.newSpec, tc.expectNewSpec) {
```

Comment:
Can you change `reflect.DeepEqual` to `cmp.Equal` to make sure these tests aren't depending on private field comparison?

Reply:
@mikedanese yep, for better review, I will create a new PR for this change.

Reply:
@mikedanese 
So this `apiequality.Semantic.DeepEqual` should also be replaced ? https://github.com/kubernetes/kubernetes/blob/4e63af2324b8562ac41f46a2e82b1b8a4224c048/pkg/kubelet/kubelet_node_status_test.go#L312